src/LogMonitor.js

```diff
@@ -122,17 +137,7 @@ export default class LogMonitor extends Component {
     }
   }
 
-  componentDidUpdate() {
-    const node = this.refs.container;
-    if (!node) {
-      return;
-    }
-    if (this.scrollDown) {
-      const { offsetHeight, scrollHeight } = node;
-      node.scrollTop = scrollHeight - offsetHeight;
-      this.scrollDown = false;
-    }
-  }
+  componentDidUpdate = this.scroll;
```

Comment:
If so, agree, in my case we pass `prevProps` and `prevState` to `this.scroll`, which is  unnecessary.